parser/ast.go

```diff
@@ -233,6 +262,26 @@ func (x *Map) String() string {
 
 func (x *Map) Type() Type { return MapType }
 
+func (x *Map) GetProperty(name string) (Property *Property, found bool) {
```

Comment:
Generally all exported functions should have at least a simple godoc comment, although we are not always good at following that rule.